qa/zcash/smoke_tests.py

```diff
@@ -218,47 +218,49 @@ def async_txid_cmd(results, case, zcash, name, args=[]):
             results[case] = False
     return txid
 
-def z_sendmany(results, case, zcash, from_addr, recipients):
+def z_sendmany(results, case, zcash, from_addr, recipients, privacy_policy):
     return async_txid_cmd(results, case, zcash, 'z_sendmany', [
         from_addr,
         [{
             'address': to_addr,
             'amount': amount,
         } for (to_addr, amount) in recipients],
-        1 # minconf
+        1, # minconf
+        0.00001, # fee
```

Comment:
We use `0.00001` explicitly here, but elsewhere in the script we use `DEFAULT_FEE` and assume that this value is equal to it (which it is, but it is clearer to use the constant).
```suggestion
        DEFAULT_FEE, # fee
```
(There is a small difference: `0.00001` is a `float` while `DEFAULT_FEE` is a `Decimal`. Using a `Decimal` here is correct and actually better.)